### ISSUE TITLE
vulkanmod refreshing the chunks when high render distance 

### ISSUE THREAD

**Issue by mamaecombei** [xCollateral/VulkanMod#507]
### Describe the bug

https://youtu.be/p4xBtBB1QPI 

### Steps to reproduce

render distance 32 is making my game refreshes every 1 second

### Log files

[Uploading latest.log…]()

**Comment by NXTler** [comment]
That's your GPU running out of VRAM, there is nothing we can do about it. Try closing a few programs.

**Comment by Felix14-v2** [comment]
> there is nothing we can do about it

That's not true, VulkanMod could just stop loading new chunks after reaching the memory limit like Nvidium or Voxy does.
